### PR TITLE
Add greeting to Kottans from Kasionio

### PR DESCRIPTION
**Pull request by kasionio**

### CODE REVIEW

**Review by lenadgit — CHANGES_REQUESTED** [review]
please - do not remove a code of another student 
codeline
Could you please fix it?

**Review by miroslav-kolomiets — APPROVED** [review]
Nice job!

File: README.md [hunk 1]
```diff
@@ -1935,3 +1942,4 @@ Hello, it`s [me](https://github.com/Labsik)
 
 ---
 
+
```
Comment (miroslav-kolomiets) [inline README.md:1945]: But this new line is unnecessary.

Comment (kasionio) [reply README.md:1945]: Thanks for the review! I see that line 1945 is empty now.

**Review by lenadgit — APPROVED** [review]

**Pull request merged**